packages/apiserver/src/permission/grant.guard.ts

```diff
@@ -35,6 +35,6 @@ export class GrantGuard implements CanActivate {
     if (permissions && grants.every((grant) => permissions.includes(grant)))
       return true;
```

Comment:
As I checked, it looks weird, that `permissions` is a string and we check does the string contains grant